src/ctia/events/obj_to_event.clj

```diff
@@ -0,0 +1,61 @@
+(ns ctia.events.obj-to-event
+  (:require [clojure.data :refer [diff]]
+            [schema.core :as s]
+            [ctia.lib.time :as t]
+            [ctia.schemas.actor :refer [StoredActor]]
+            [ctia.schemas.campaign :refer [StoredCampaign]]
+
+            [ctia.events.schemas :as vs]))
+
+
+(s/defn to-create-event :- vs/CreateEvent
+  [model :-  {s/Any s/Any}
```

Comment:
I think that the '{s/Any s/Any}' schemas are a smell.  It that tells me that you needn't bother with schemas at all in this namespace.